generals/game.py

```diff
@@ -138,7 +109,7 @@ def action_mask(self, agent: str) -> np.ndarray:
 
             # check if destination is road
             passable_cell_indices = (
-                self.channels["passable"][destinations[:, 0], destinations[:, 1]] == 1.0
+                self.channels.passable[destinations[:, 0], destinations[:, 1]] == 1.0
```

Comment:
Since `passable` property is boolean, isn't it weird to compare it with `1.0`?

Similar thing happens also at other places.

Reply:
`== 1.0` is indeed weird, but I like `==0` or `==1`, my brain realizes what is going on quicker than putting nothing. Another option is `True`/`False`, but thats not so elegant (imo :D) wdyt ? @Puckoland @jdujava

Reply:
Alright, `is True` is not a viable option because `numpy` uses `numpy._True` instead of `True`

Reply:
Pucko found out that `is True` doesnt work for numpy. So the option is to choose either nothing or `== True` which I strongly dislike. These arrays are assigned `1` or `0` in multiple places so I feel like doing it also in comparison is a good way.